src/backend/distributed/transaction/remote_transaction.c

```diff
@@ -32,12 +33,17 @@ static void WarnAboutLeakedPreparedTransaction(MultiConnection *connection, bool
 
 /*
  * StartRemoteTransactionBeging initiates beginning the remote transaction in
- * a non-blocking manner.
+ * a non-blocking manner. The function sends "BEGIN" followed by a function call
+ * (i.e., assign_distributed_transaction_id()) to assign the distributed transaction
+ * id on the remote node.
```

Comment:
I'd just remove the "followed by a function call" and unparenthesize the actual function - no info added.